ios/MaasNearbyTransit/Sources/MaasNearbyTransit/Compositions/NearbyTransitView.swift

```diff
@@ -0,0 +1,60 @@
+import SwiftUI
+
+public struct NearbyTransitView: View {
+
+    let iitems = [
+        NearbyTransitComponentsPreviewData.stopWithSchedulesWithDepartures,
+        NearbyTransitComponentsPreviewData.stopWithSchedulesWithDepartures,
+        NearbyTransitComponentsPreviewData.stopWithSchedulesWithDepartures,
+        NearbyTransitComponentsPreviewData.stopWithSchedulesWithDepartures,
+        NearbyTransitComponentsPreviewData.stopWithSchedulesWithDepartures,
+        NearbyTransitComponentsPreviewData.stopWithSchedulesWithDepartures,
+        NearbyTransitComponentsPreviewData.stopWithSchedulesWithDepartures,
+        NearbyTransitComponentsPreviewData.stopWithSchedulesWithDepartures,
+    ]
+
+    public init() {}
+
+    public var body: some View {
+        VStack {
+            filterView()
+            scrollView()
+        }
+    }
+}
+
+private extension NearbyTransitView {
```

Comment:
I think extension is unnecessary 🤔 
```suggestion
```

Reply:
All `func()` that are not marked as `public` should be marker as `private`, but why use multiple `private func` when we could have a one `private extension`, or you talking about that we don't need define non `public func` as `private` ? 🤔